fake_ubersmith/api/administrative_local.py

```diff
@@ -17,8 +17,8 @@
 
 
 class AdministrativeLocal(Base):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, data_store):
+        super(AdministrativeLocal, self).__init__(data_store)
```

Comment:
Python 3 does not need the calling class in super anymore, you can just do: 

super().__init__(data_store)